### PR TITLE
FEAT: Added List,Modal,Rating,Toast components

### PR DESCRIPTION
**Pull request by kotesh-arya**
Screenshot from 2022-03-28 22-38-23
Screenshot from 2022-03-28 22-38-17
Screenshot from 2022-03-28 22-37-59
Screenshot from 2022-03-28 22-37-48

### CODE REVIEW

**Review by vinc3nati — APPROVED** [review]
Looks good, but don't define `:root` in every css file rather define it in one common parent file and use it everywhere.

**Comment by kotesh-arya** [comment]
Will do that 👍️

File: Components/Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
+:root {
+    --success: #008000;
+    --warning: #FFA500;
+    --error: #FF0000;
+    --medium-font-size: 1.5rem;
+}
+
+
```
Comment (Adarsh-Patel-Dev) [inline Components/Modal/modal.css]: REMOVE extra spaces here

**Review by Adarsh-Patel-Dev — APPROVED** [review]

File: Components/Modal/modal.css (hunk 1 above)
Comment (kotesh-arya) [reply Components/Modal/modal.css]: Sure, will update that

**Pull request merged**